### DISCUSSION TITLE
148 breaks configuration

### DISCUSSION THREAD

**Discussion by NewbieOnPractice** [Rem0o/FanControl.Releases#1577] [General]
Hi, after updating to v148 FanControl doesn't find speed controls/sensors and my configuration is basically broken. Refreshing doesn't help to bring them back. I reverted to v147 to make it work again. I'll attach a couple of screenshot below. Maybe they're helpful for the analysis. Any idea?

Boot-up after 148 update:
![fancontrol_failed_to_initialize](https://user-images.githubusercontent.com/1999112/221983847-d05edcd1-1c42-4085-8ec5-a14eeff8e315.png)

Error messages 1+2:
![fancontrol_errors_01](https://user-images.githubusercontent.com/1999112/221984042-8105222d-3106-47ce-a1b6-53ca3a3d8687.png)
![fancontrol_errors_02](https://user-images.githubusercontent.com/1999112/221984089-7fd5fdab-eac8-4ef4-8260-3db9ca705786.png)

Broken configuration (148):
![fancontrol_v148](https://user-images.githubusercontent.com/1999112/221984173-ab622d09-848d-4c0f-aa41-58e948bac241.png)

Healthy configuration (147):
![fanspeed_config_147](https://user-images.githubusercontent.com/1999112/221984394-dae01b73-0cef-4343-995d-7c640043049e.png)

**Comment by Rem0o** [comment]
Probably linked to this?

https://github.com/LibreHardwareMonitor/LibreHardwareMonitor/commit/af0f4b696bb48074305153773f4ec3e633e68604#diff-ab954123283065b240cec0d9311bd2feffade5803136f44cfe7d89242a81373a

**Comment by NewbieOnPractice** [comment]
Probably. Running the lib that came with 147 with 148 works. Interstingly, the version no. of the LHM lib hasn't changed (0.9.1). I'll post there tomorrow.

**Comment by EvanMulawski** [comment]
I'm seeing the same thing - and plugins were not detected/loaded. Rolling back to v147 resolved this.

**Reply by Rem0o** [reply]
Oh boy I'm in for a treat.  The updater gets downloaded in the configuration folder, then extracts the zip in there, which creates a mess of a nested fancontrol.  I uploaded a hotfix as fast as I could. Uploaded a new updater.exe that will attempt to fix this mess I just made 😅

**Comment by NewbieOnPractice** [comment]
https://github.com/LibreHardwareMonitor/LibreHardwareMonitor/issues/990
